terraform/install.sh

```diff
@@ -249,7 +275,7 @@ installTerraform
 # Provision Stackdriver Sandbox cluster
 getProject;
 applyTerraform;
+installMonitoring || true;
```

Comment:
What's the reason for the `|| true`? Won't the script stop early within the function anyway if it encounters an error?

Reply:
If there's an error with the Terraform apply like an API returns a 403 then the entire script will stop - this lets the rest of the script continue even if the monitoring isn't created.

Reply:
Please add a comment on this above the line, but I agree with the logic for the upcoming release.

Reply:
Have you tested that it works? My assumption would have been that the script would exit right at line 199 if it encountered an error. I'd think you'd have to add `|| true` there

Reply:
Yep it works! I tested it with a project w/o a workspace with and without the || true. This screenshot is with it[](url)
![Screenshot 2020-08-04 at 1 41 22 PM - Display 2](https://user-images.githubusercontent.com/33471452/89343114-a744ee00-d658-11ea-9832-8a25315e1246.png)